recipes/google-weather-tools/meta.yaml

```
@@ -0,0 +1,85 @@
+{% set name = "google-weather-tools" %}
+{% set version = "0.3.2" %}
+{% set git_rev = "conda-candidate-v0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/google/weather-tools.git
+  git_rev: "{{ git_rev }}"
```

If all you want is to just get a git tag, use:
```suggestion
  url: https://github.com/google/weather-tools/archive/refs/tags/v{{ version }}.tar.gz
  sha256: YOURSHAHERE
```

---

Thanks! Will TAL at this soon.